### PR TITLE
update branch to main

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

File: .ci/jobs/defaults.yml [hunk 1]
```diff
@@ -50,7 +50,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
-    periodic-folder-trigger: 1w
```
Comment (cachedout) [inline .ci/jobs/defaults.yml:53]: Seems unrelated to this change.

File: .ci/jobs/defaults.yml [hunk 2]
```diff
@@ -50,7 +50,6 @@
         timeout: '15'
         use-author: true
         wipe-workspace: 'True'
```
Comment (v1v) [inline .ci/jobs/defaults.yml:52]:
```suggestion
        wipe-workspace: 'True'
    periodic-folder-trigger: 1w
```

**Review by cachedout — APPROVED** [review]

**Review by axw — APPROVED** [review]

**Review by simitt — APPROVED** [review]
